app/kumactl/cmd/get/get_meshes.go

```diff
@@ -58,13 +57,8 @@ func printMeshes(meshes []*mesh.MeshResource, out io.Writer) error {
 				mesh := meshes[i]
 
 				mtls := "off"
-				if mesh.Spec.Mtls.GetEnabled() {
-					switch mesh.Spec.GetMtls().GetCa().GetType().(type) {
-					case *mesh_proto.CertificateAuthority_Provided_:
-						mtls = "provided"
-					case *mesh_proto.CertificateAuthority_Builtin_:
-						mtls = "builtin"
-					}
+				if mesh.MTLSEnabled() {
+					mtls = mesh.Spec.GetMtls().GetEnabledBackend()
```

Comment:
Question to @subnetmarco how should `kumactl get meshes` look like?
Right now given
```
type: Mesh
name: default
mtls:
  enabledBackend: ca-1
  backends:
  - name: ca-1
    type: builtin
  - name: ca-2
    type: provided
    config: ...
```
I print
```
NAME      mTLS  METRICS  LOGGING  TRACING
default   ca-1  off      off      off
```
If `enabledBackend` is not specified, I print `off`.

Ilya's argument is that it less information because previously we've seen the type (only the type since it was 1 CA per mesh)

Reply:
We came into conclusion with Marco that we will only display current backend with a format like you proposed which is `type/name`.
I will refactor also others in the same way when I unify the config for Tracing/Logging/Metrics.